packages/blockchain-wallet-v4-frontend/src/data/wallet/sagas.js

```diff
@@ -52,10 +53,20 @@ export default ({ coreSagas }) => {
   const editBtcAccountLabel = function * (action) {
     try {
       let { index, label } = action.payload
+      const wallets = (yield select(selectors.core.common.btc.getHDAccounts))
+        .getOrFail()
+        .map(wallet => wallet.label)
+      const isUnique = (value, allValues) => {
+        const walletIdx = wallets.indexOf(value)
+        return walletIdx !== index && walletIdx > -1
+          ? 'Wallet name is already taken.'
+          : undefined
+      }
```

Comment:
to reduce redundancy, lets rename to be `requireUniqueWalletName` and then export it from
`packages/blockchain-wallet-v4-frontend/src/services/FormHelper/validators.js` and import into both sagas.